### DISCUSSION TITLE
Sending a file more than 15mb via API Postal

### DISCUSSION THREAD

**Discussion by alekseystepanov92** [postalserver/postal#2216] [Development discussions]
When trying to transfer a file("attachments") over 15mb via api (base 64). 
API answers {'status': 'internal-server-error'}.
is it possible to send a file larger than 15mb?

**Comment by casualmatt** [comment]
Change to the API will be limited, for now, Postal plans to migrate to a new API backend, at least this assumption is base on their comment on the API documentation --> https://docs.postalserver.io/developer/api

> This API does not support managing all the functions of Postal. There are plans to introduce a new v2 API which will have more functionality and significantly better documentation. We do not have an ETA for this. Additionally, we will not be accepting any pull requests to extend the current API to have any further functionality than it currently does.

**Comment by willpower232** [comment]
Have a look at #337, it is likely that you have a problem with the database handling all that information in one go.
